docs/mkdocs/driver_containers.md

```diff
@@ -0,0 +1,12 @@
+# How to build driver-containers to be used by KMMO
+
+### Pre-build driver-containers
+
+When a `Module` CR is applied to the cluster, KMMO will create a DaemonSet to run the driver-container specified in it.
+
+In order for driver-container that depends on in-tree kernel-modules to work, KMMO is mounting `/lib/modules/$(KVER)` from the node to the container, which means that it will override all files located in `/lib/modules/$(KVER)` inside the driver-container.
```

Comment:
```suggestion
In order for driver-containers that depend on in-tree kernel-modules to work, KMMO is mounting `/lib/modules/${KVER}` from the node to the container, which means that it will override all files located in `/lib/modules/${KVER}` inside the driver-container.
```